frinx-uniconfig/user-guide/uniconfig-operations/templates-manager/rpc_create-multiple-templates/readme.md

```diff
@@ -0,0 +1,262 @@
+        ]
+      },
+      {
+        "tag": "template-tags:create",
+        "paths": [
+          "/services"
+        ]
+      }
+    ]
+  }
+}
+```
+
+```RPC response, Status: 200
+```
+
+### Failed Example
+
+Failed to find YANG schema repository.
+
+```bash RPC Request
+curl --location --request POST 'http://127.0.0.1:8181/rests/operations/template-manager:create-multiple-templates' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+        "templates": [
+            {
+                "template-name": "Template-name1",
+                "yang-repository": "schema-1",
+                "template-configuration": {
+                    "system:system-config": {
+                        "@": {
+                            "template-tags:operation": "replace"
+                        },
+                        "timezone": "CEST"
+                    }
+                }
+            },
+            {
+                "template-name": "Template-name2",
+                "yang-repository": "schema-2",
+                "template-configuration": {
+                    "system:system-config": {
+                        "@": {
+                            "template-tags:operation": "replace"
+                        },
+                        "timezone": "CEST"
+                    }
+                }
+            }
+        ]
+    }
+}'
+```
+
+```json RPC Response, Status: 404
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "uniconfig-error",
+        "error-tag": "missing-element",
+        "error-message": "Failed to find schema repository: schema-1",
+        "error-info": {
+          "node-id": "Template-name1"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Failed Example
+
+Failed to parse template configuration.
+
+```bash RPC Request
+curl --location --request POST 'http://127.0.0.1:8181/rests/operations/template-manager:create-multiple-templates' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+        "templates": [
+            {
+                "template-name": "Template-name1",
+                "yang-repository": "schema-1",
+                "template-configuration": {
+                    "system:system-config": {
+                        "@": {
+                            "template-tags:operation": "replace"
+                        },
+                        "timezone": "CEST"
+                    }
+                }
+            },
+            {
+                "template-name": "Template-name2",
+                "yang-repository": "schema-2",
+                "template-configuration": {
+                    "dhcp:dhcp": {
+                        "enabled": true
+                    },
+                    "service": ["dhcp"]
+                }
+            }
+        ]
+    }
+}'
+```
+
+```json RPC Response, Status: 500
+{
+  "errors": {
+    "error": [
+      {
+        "error-type": "application",
+        "error-tag": "operation-failed",
```

Comment:
this is user fault - user provided invalid data

Reply:
Done.

Reply:
Done.